cpp/ql/src/Security/CWE/CWE-295/SSLResultConflation.ql

```diff
@@ -31,6 +31,14 @@ module VerifyResultConfig implements DataFlow::ConfigSig {
   predicate isSink(DataFlow::Node sink) {
     exists(GuardCondition guard | guard.getAChild*() = sink.asExpr())
   }
+
+  predicate observeDiffInformedIncrementalMode() { any() }
```

Comment:
Even though this is a query with two dataflow computations, I think it is ok to make it diff-informed, as it is conceptually the conjunction of two single-dataflow queries.